docs/content/plugins/discoverability.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Plugin Discoverability
+description: Learn more about searching and broadcasting available plugins for Porter
+---
+
+## Search
+
+Porter maintains a list of plugins that users can search via
+`porter plugin search [NAME]`. If no name is supplied, the full listing will be
+returned.  See the help menu for all command options: `porter plugin search -h`.
```

Comment:
How about instead of telling them to run the help command, let's link them to the help text that is generated and hosted on the site. So when we mention the command above, link it to `/cli/porter_plugin_search/`